### ISSUE TITLE
LoadURL showing text/plain instead of text/html on Android

### ISSUE THREAD

**Issue by afonsolage** [gree/unity-webview#184]
I'm having an issue when building to APK. The WebView works fine on MAC OS Unity Editor, it displays everything as expected, but when I build an APK, it just shows the page as text/plain instead of text/html.

I'm using my own MainActivity, but I'm overriding CUnityPlayerActivity:
```
public class MainActivity extends CUnityPlayerActivity {

	@Override
	public void onCreate(Bundle bundle) {
		super.onCreate(bundle);
	}
```

And I'm loading the URL as follow:

```
private void CreateBrowser(string URL)
    {
        if (go != null)
        {
            Close();
        }

        go = new GameObject("WebViewObject");
        go.layer = LayerMask.NameToLayer("MYUI");

        var collider = go.AddComponent<BoxCollider>();
        collider.size = new Vector3(10, 10, 0);
        collider.center = new Vector3(0, 0, -10);
        collider.isTrigger = true;

        webViewObject = go.AddComponent<WebViewObject>();
        webViewObject.Init(
            cb: (msg) =>
            {
                if (msg == "close")
                {
                    MyCodeLogin.CloseWebPanel();
                }
                else if (msg.StartsWith("error:"))
                {
                    msg = msg.Substring(6);
                    msg = WWW.UnEscapeURL(msg);
                    MYDebug.LogWarning("JS Error: " + msg);
                }
                else if (msg.StartsWith("log:"))
                {
                    msg = msg.Substring(4);
                    msg = WWW.UnEscapeURL(msg);
                    MYDebug.Log("JS Log: " + msg);
                }
            },
            err: (msg) =>
            {
                MYDebug.Log(string.Format("CallOnError[{0}]", msg));
            },
            ld: (msg) =>
            {
                MYDebug.Log(string.Format("Web Page loaded! {0}", msg));
            },
            enableWKWebView: true
        );
        webViewObject.AddCustomHeader("Auth-Code", "asddsaasd");

#if DEBUG
        webViewObject.AddCustomHeader("Cache-Control", "no-store, must-revalidate");
        webViewObject.AddCustomHeader("Expires", "0");
#endif

        webViewObject.SetMargins(0, 0, 0, 0);
        webViewObject.SetVisibility(true);
        webViewObject.LoadURL("http://www.google.com");

        MYDebug.Log("Loading URL: " + URL);
    }
```
But even so, when I open the UnityWebView it just shows me the source code of the page (in that case google.com)

![screenshot_3](https://cloud.githubusercontent.com/assets/1176452/26473536/667c1d6c-4182-11e7-9978-c209874d4daf.jpg)

on my AndroidManifest.xml, I'm using:

```
[..]
    <uses-sdk
        android:minSdkVersion="9"
        android:targetSdkVersion="18" />

[..]
android:hardwareAccelerated="true"
```

Any clues?

**Comment by afonsolage** [comment]
Using:

Unity 5.2.4f1
Android 4.4.2
NoxPlayer

**Comment by KojiNakamaru** [comment]
Thank you for the report. I've investigated the issue and fixed it in #185. Could you please try the latest?

**Comment by afonsolage** [comment]
Thnaks! It's working now!!
